# This message holds all joint params and settings
#
# This is a helper type and using as a part of other messages

float32 lower_limit
float32 upper_limit
float32 offset
float32 reverse
TypeJointMode mode